docs/faq/terminology.rst

```diff
@@ -0,0 +1,46 @@
+.. _Observation_Terminology:
+
+***********
+Terminology
+***********
+
+Scheduling
+==========
+
+Scheduling observations is a common task in astronomy, but there is a lack of
+formal terminology. Here we define the terminology as it will be used in astroplan
+for scheduling tasks. This is for consistency within the code and the documentation.
+
+The terms used are as follows:
+
+* **visit**: a period of time spent observing a single target
+* **observing block** (OB): the minimum schedulable block (tentative: a group of visits)
```

Comment:
I don't quite understand ``(tentative: a group of visits)``?  A would think an OB is always at least a visit or *smaller*: for example, you might want to observe the same target in two different filters - in that case there are two OBs but just one visit